operations/deployment/terraform/modules/02_networking.tf

```diff
@@ -85,7 +84,7 @@ resource "aws_security_group" "allow_http" {
 resource "aws_security_group" "allow_https" {
  name        = "allow_https"
```

Comment:
BTW the naming is too generic to be re-used in some existing VPC that might lead to conflicts.
Something like `st2_allow_https` might work better

Reply:
captured here: https://github.com/bitovi/github-actions-deploy-stackstorm/issues/8